pkg/gadgets/common/mntns_filter.h

```diff
@@ -0,0 +1,33 @@
+/* SPDX-License-Identifier: (GPL-2.0 WITH Linux-syscall-note) OR Apache-2.0 */
+
+#ifndef MNTNS_FILTER_H
+#define MNTNS_FILTER_H
+
+#include <bpf/bpf_helpers.h>
+#include <bpf/bpf_core_read.h>
+#include <bpf/bpf_tracing.h>
+
+const volatile bool filter_by_mnt_ns = false;
```

Comment:
Rename to `ig_filter_by_mnt_ns`?

Reply:
Renamed the variables, functions and map declarations to have a `gadget_` prefix as done in the socket enricher.